content/lang/collections/loans/loan_langs.en.php

```diff
@@ -60,7 +60,7 @@
 $LANG['ADD_NEW_INST'] = 'Add a New Institution';
 $LANG['CREATE_EXCHANGE'] = 'Create Exchange';
 $LANG['TRANS_BY_INST'] = 'Transaction Records by Institution';
-$LANG['OUTGOING_EX_SENT'] = 'Outgoing exchange; Sent ';
+$LANG['BARCODE_CATNUM'] = 'Outgoing exchange; Sent ';
```

Comment:
It looks like BARCODE_CATNUM occurrs twice in this file. One of them needs to re-named, and the new name should be implemented globally.